src/ContainerManagement.js

```diff
@@ -0,0 +1,79 @@
+import Dependency from "./Dependency"
+
+const rall = /.+/
+
+export default {
```

Comment:
I'm treating this file more like a mixin/interface than it's own class (since JS doesn't have multiple inheritance). We were faking it by looping over everything and calling the different constructors, but ES6 prohibits that. `this.dependencies` is instead initialized in the single constructor in [Injector.js](https://github.com/timmywil/spur-ioc/blob/babel/src/Injector.js#L14). Had I written this in ES6 originally, I probably would have encapsulated purposes and done things like `this.containerManager.method`, rather than inheriting, but I wasn't sure what that would break and this was just the first step.